src/utils/get-organized-arguments.ts

```diff
@@ -287,12 +287,21 @@ export default async function getOrganizedArguments(): Promise<OrganizedArgument
 
 		// Validate data, if necessary
 		if (mergedSpec.data.accepts && mergedSpec.data.accepts instanceof Array) {
-			// @ts-expect-error: TypeScript is confused here...
-			if (!mergedSpec.data.accepts.includes(organizedArguments.data)) {
+			// Find unrecognized data by removing acceptable values
+			let remnants = ` ${organizedArguments.data} `;
+
+			for (const current of mergedSpec.data.accepts) {
+				remnants = remnants.replace(` ${current} `, ' ');
```

Comment:
Couldn't you remove it entirely, instead of replacing it with a single space? That would make it so you don't have to trim afterwards as well, I think.

Reply:
Well, sticking with the idea of having the single string that's being used to contain the remaining matchable items.... if there are multiple words, and the middle of 3 words is removed (including spaces), then the remaining 2 words will no longer be separated by a space, causing subsequent match attempts to fail.

However, if all the values are split into arrays, then possibly it could be done without having guard spaces at all.

Reply:
I see what you mean that makes sense, I wouldn't split it into an array if you don't have to, especially regarding the edge cases you mentioned.